src/yuzu/configuration/configure_motion_touch.cpp

```diff
@@ -187,13 +184,58 @@ void ConfigureMotionTouch::ConnectEvents() {
     });
 }
 
+void ConfigureMotionTouch::onUDPAddServer() {
+    QRegExp re(tr("^(?:(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.){3}(?:25[0-5]|2[0-4][0-9]|[01]?["
+                  "0-9][0-9]?)$")); // a valid ip address
+    bool ok;
+    QString port_text = ui->udp_port->text();
+    QString server_text = ui->udp_server->text();
+    QString server_string = server_text + tr(":") + port_text;
+    int port_number = port_text.toInt(&ok, 10);
+    int row = udp_server_list_model->rowCount();
+
+    if (!ok) {
+        QMessageBox::warning(this, tr("Yuzu"), tr("Port number has invalid characters"));
```

Comment:
```suggestion
        QMessageBox::warning(this, tr("yuzu"), tr("Port number has invalid characters"));
```

Ditto for other occurrences.